Source/Protected.swift

```diff
@@ -40,38 +94,26 @@ final class UnfairLock {
         unfairLock.deallocate()
     }
 
-    private func lock() {
+    fileprivate func lock() {
         os_unfair_lock_lock(unfairLock)
     }
 
-    private func unlock() {
+    fileprivate func unlock() {
         os_unfair_lock_unlock(unfairLock)
     }
-
-    /// Executes a closure returning a value while acquiring the lock.
-    ///
-    /// - Parameter closure: The closure to run.
-    ///
-    /// - Returns:           The value the closure generated.
-    func around<T>(_ closure: () -> T) -> T {
-        lock(); defer { unlock() }
-        return closure()
-    }
-
-    /// Execute a closure while acquiring the lock.
-    ///
-    /// - Parameter closure: The closure to run.
-    func around(_ closure: () -> Void) {
-        lock(); defer { unlock() }
-        return closure()
-    }
 }
 
+#endif
+
 /// A thread-safe wrapper around a value.
 @propertyWrapper
 @dynamicMemberLookup
 final class Protected<T> {
-    private let lock = UnfairLock()
+    #if os(Linux)
+    private let lock: Lock = MutexLock()
+    #else
+    private let lock: Lock = UnfairLock()
+    #endif
```

Comment:
As before, doing this without calling out Linux specifically would look like this:
```suggestion
    #if os(macOS) || os(iOS) || os(watchOS) || os(tvOS)
    private let lock = UnfairLock()
    #else
    private let lock = MutexLock()
    #endif
```